learn/cookbooks/docker.md

```diff
@@ -0,0 +1,123 @@
+:::
+
+## Run Meilisearch
+
+After running `docker pull` and downloading the Meilisearch image, use `docker run` to launch it:
+
+```sh
+docker run -it --rm \
+  -p 7700:7700 \
+  -v $(pwd)/meili_data:/meili_data \
+  getmeili/meilisearch:v0.28
+```
+
+### Configure Meilisearch
+
+Meilisearch accepts a number of instance options during launch. You can configure instance options in two ways: environment variables and CLI arguments. Some options are only available as CLI arguments—[consult our configuration reference for an exhaustive list](/learn/configuration/instance_options.md).
+
+#### Passing instance options with environment variables
+
+To pass environment variables to Docker, you must pass the `-e` argument to `docker run`. The example below launches Meilisearch with a master key:
+
+```sh
+docker run -it --rm \
+  -p 7700:7700 \
+  -e MEILI_MASTER_KEY='MASTER_KEY'\
+  -v $(pwd)/meili_data:/meili_data \
+  getmeili/meilisearch:v0.28
+```
+
+#### Passing instance options with CLI arguments
+
+If you want to pass command-line arguments to Meilisearch with Docker, you must explicitly run the `meilisearch` binary after indicating the Meilisearch image you want to use:
+
+```sh
+docker run -it --rm \
+  -p 7700:7700 \
+  -v $(pwd)/meili_data:/meili_data \
+  getmeili/meilisearch:v0.28
+  meilisearch --master-key="MASTER_KEY"
+```
+
+## Managing data
+
+When using Docker, your working directory is `/meili_data`. This means the location of your database file is `/meili_data/data.ms`.
+
+### Data persistency
+
+By default, data written to a Docker container is deleted every time the container stops running. This data includes your indexes and the documents they store.
+
+To keep your data intact between reboots, specify a dedicated volume by running Docker with the `-v` command-line option:
+
+```sh
+docker run -it --rm \
+  -p 7700:7700 \
+  -v $(pwd)/meili_data:/meili_data \
+  getmeili/meilisearch:v0.28
+```
+
+The example above uses `$(pwd)/meili_data`, which is a directory in the host machine. Depending on your OS, mounting volumes from the host to the container might result in performance loss and is only recommended when developing and prototyping your application.
+
+### Generating dumps and updating Meilisearch
+
+To export a dump, [use the dumps endpoint as described in our dedicated dumps guide](/learn/advanced/dumps.md). Once the task is complete, you can access the dump file in `/meili_data/dumps` inside the volume you configured with `-v`.
+
+To import a dump, use the `--import-dump` command-line option and specify the path to the dump file. Make sure the path points to a volume reachable by Docker:
+
+```sh
+docker run -it --rm \
+  -p 7700:7700 \
+  -v $(pwd)/meili_data:/meili_data \
+  getmeili/meilisearch:v0.28
+  meilisearch --import-dump /meili_data/dumps/20200813-042312213.dump
+```
+
+::: warning
+If you are using a shared volume, you must delete `/meili_data/data.ms` before importing a dump.
```

Comment:
Is there any link we could provide to elaborate on the meaning of this? Putting myself in the shoes of a user, I've got no clue what a shared volume is or how to know if this applies to me.

Reply:
I pretty much copied this verbatim from an SME. As far as I understand, the shared volume is the external data volume. Updated the wording so this is clear.